sdk/cosmos/azure-cosmos/azure/cosmos/aio/_database.py

```diff
@@ -177,15 +178,61 @@ async def create_container(
         vector_embedding_policy: Optional[Dict[str, Any]] = None,
         change_feed_policy: Optional[Dict[str, Any]] = None,
         full_text_policy: Optional[Dict[str, Any]] = None,
+        return_properties: Literal[False],
         **kwargs: Any
     ) -> ContainerProxy:
+        ...
+
+    @overload
+    async def create_container(
+            self,
+            id: str,
+            partition_key: PartitionKey,
+            *,
+            indexing_policy: Optional[Dict[str, str]] = None,
+            default_ttl: Optional[int] = None,
+            offer_throughput: Optional[Union[int, ThroughputProperties]] = None,
+            unique_key_policy: Optional[Dict[str, str]] = None,
+            conflict_resolution_policy: Optional[Dict[str, str]] = None,
+            initial_headers: Optional[Dict[str, str]] = None,
+            computed_properties: Optional[List[Dict[str, str]]] = None,
+            analytical_storage_ttl: Optional[int] = None,
+            vector_embedding_policy: Optional[Dict[str, Any]] = None,
+            change_feed_policy: Optional[Dict[str, Any]] = None,
+            full_text_policy: Optional[Dict[str, Any]] = None,
+            return_properties: Literal[True],
+            **kwargs: Any
+    ) -> tuple[ContainerProxy, CosmosDict]:
```

Comment:
Not blocking for this PR - but something to consider before these changes go GA is whether there's a scenario where customers want to call the create/replace APIs _without_ instantiating a `Proxy` object (i.e. something more light-weight).